opentracing-cdi/src/main/java/io/opentracing/contrib/cdi/internal/TracerInitializer.java

```diff
@@ -0,0 +1,60 @@
+package io.opentracing.contrib.cdi.internal;
+
+import io.opentracing.Tracer;
+import io.opentracing.contrib.tracerresolver.TracerResolver;
+import io.opentracing.util.GlobalTracer;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.context.Initialized;
+import javax.enterprise.event.Observes;
+import javax.enterprise.inject.Instance;
+import javax.inject.Inject;
+import java.util.logging.Logger;
+
+/**
+ * This is not part of the "business" of this example, but can be used as an example of how to register a Tracer
+ * with the GlobalTracer. On some platforms, like Wildfly Swarm, this is done automatically for you.
+ *
+ * @author Juraci Paixão Kröhling
+ */
+public class TracerInitializer {
```

Comment:
I think there might be a problem when multiple instrumentations are being used together and all of them are using tracerresolver. 

It would be maybe better if the tracer artifact which supports tracerresolver would resolve tracer and register it to the GlobalTracer. Instrumentations would just depend on GlobalTracer.